demo/run.sh

```diff
@@ -60,30 +59,43 @@ function createHostFactoryToken() {
     client \
     /conjurinc/ansible/create_host_token.sh
 
-  echo '-----'
+  export HFTOKEN=$(<conjur-client-files/output/hftoken.txt)
 }
 
-function runAnsible() {
-  echo "Running Ansible playbook"
+function conjurizeTargetContainer() {
+  echo '-----'
+  echo "Conjurizing the target container with Ansible"
   echo '-----'
 
-  export HFTOKEN=$(<client-output/hftoken.txt)
-
-  ansible-playbook demo.yml
+  ansible-playbook conjurize-container.yml
+}
 
+function RetrieveSecretInTargetWithCli() {
+  echo '-----'
+  echo "Retrieving secret in target container with Conjur cli"
   echo '-----'
+
+  docker exec ansible-target bash -c /conjurinc/ansible/conjur_cli.sh
```

Comment:
`docker exec` is a code smell for fighting against Docker's immutability.

I would like to see `docker-compose run --rm ansible-role-conjur-test conjur_cli.sh` here instead.

Reply:
Worked in this for ~ an hour and couldn't get it going so I'll work on it after the whole demo works